silverbackis/common-js-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "Silverback\\CommonJsBundle\\SilverbackCommonJsBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
```

Comment:
This is not needed anymore

Reply:
Both the "bundles" and "copy-from-recipe" keys?

Reply:
Sorry, stupid question. Removed both - the config dir was not required.